Module/uti/byysk_armor.uti.json

```diff
@@ -2,7 +2,7 @@
   "__data_type": "UTI ",
   "AddCost": {
     "type": "dword",
-    "value": 0
+    "value": 375
```

Comment:
Same comment about agent gear applies here too. It looks like a static 375 was set but please ensure the numbers are within the T3/T4 range.

Reply:
"All prices have been set to 2.5x in the files to reflect the 0.4x modifier that vendors buy items at."

40% of 375 is 150, which is the correct buying price for a tier 3 loot item according to the table above that you okay'd. IE: the player will only receive 150 credits for selling this item.